geopandas/geodataframe.py

```diff
@@ -1456,29 +1483,45 @@ def explode(self, column=None, **kwargs):
             column = self.geometry.name
         # If the specified column is not a geometry dtype use pandas explode
         if not isinstance(self[column].dtype, GeometryDtype):
-            return super(GeoDataFrame, self).explode(column, **kwargs)
-            # TODO: make sure index behaviour is consistent
+            if compat.PANDAS_GE_11:
+                return super(GeoDataFrame, self).explode(column, ignore_index)
```

Comment:
```suggestion
                return super().explode(column, ignore_index=ignore_index)
```

Better be explicit and compat with #1885